integration-tests-cypress/support/selectors.ts

```diff
@@ -0,0 +1,33 @@
+/* eslint-disable @typescript-eslint/no-use-before-define */
+import Loggable = Cypress.Loggable;
+import Timeoutable = Cypress.Timeoutable;
+import Withinable = Cypress.Withinable;
+import Shadow = Cypress.Shadow;
+
+export {};
+declare global {
+  namespace Cypress {
+    interface Chainable<Subject> {
+      byTestID(
+        selector: string,
+        options?: Partial<Loggable & Timeoutable & Withinable & Shadow>,
+      ): Chainable<Element>;
+      byTestActionID(selector: string): Chainable<Element>;
+      byLegacyTestID(selector: string): Chainable<Element>;
+    }
+  }
+}
+
+// any command added below, must be added to global Cypress interface above
+
+Cypress.Commands.add(
+  'byTestID',
```

Comment:
I would be inclined to remove this and just use `[data-test="my-id"]` where we need to in the template repo. Then we don't need the selectors file at all.